### PR TITLE
Add Ids to ease tests

### PR DESCRIPTION
**Pull request by nixocio**
Add Ids to ease tests.

See: https://github.com/ansible/awx/issues/8680#issuecomment-893663314

### CODE REVIEW

File: awx/ui/src/screens/User/UserTokenList/UserTokenList.js [hunk 1]
```diff
@@ -170,6 +170,7 @@ function UserTokenList() {
                   ]
                 : []),
               <ToolbarDeleteButton
+                ouiaId="delete-user-tokens-button"
```
Comment (AlexSCorey) [inline awx/ui/src/screens/User/UserTokenList/UserTokenList.js]: can we rename this to be user-token-delete-button.  Current phrasing suggests it delete the button itseld
```suggestion
                ouiaId="user-token-delete-button"
```

File: awx/ui/src/components/PaginatedTable/ToolbarDeleteButton.js [hunk 2]
```diff
@@ -284,7 +285,7 @@ function ToolbarDeleteButton({
         >
           <div>{t`This action will delete the following:`}</div>
           {itemsToDelete.map((item) => (
-            <span key={item.id}>
+            <span key={item.id} id={`item-to-be-delete-${item.id}`}>
```
Comment (AlexSCorey) [inline awx/ui/src/components/PaginatedTable/ToolbarDeleteButton.js]:
```suggestion
            <span key={item.id} id={`item-to-be-deleted-${item.id}`}>
```

**Review by AlexSCorey — CHANGES_REQUESTED** [review]
Super small nit picky things here.

**Review by akus062381 — APPROVED** [review]

**Review by AlexSCorey — APPROVED** [review]
Well done!

**Pull request merged**